InjectionByExample/ContainerBuilder.cs

```diff
@@ -24,7 +22,7 @@ public Container Build()
         {
             // we use reflection to get the first constructor and see what parameters are needed
             // to create an instance. 
-            // run this once, multithreaded, and store the conclusion in the registratoin.
+            // run this once, multithreaded, and store the conclusion in the registration.
```

Comment:
Thanks :D

Reply:
You're welcome. Spell checker helps a lot :)